### DISCUSSION TITLE
Make it possible to disable renovate/stability-days status check when using `internalChecksFilter: 'strict'`

### DISCUSSION THREAD

**Discussion by fabb** [renovatebot/renovate#28283] [Suggest an Idea]
### Tell us more.

We are using this config in Gitlab:
```
stabilityDays: 3,
internalChecksFilter: 'strict',
```

Since a recent renovate version update, we get a `renovate/stability-days` pipeline in our merge requests which overwrites the regular pipeline and makes our "merge requests can only be merged when our regular pipeline succeeds" setting to break.

=> we'd like to disable this `renovate/stability-days` check since we don't need it when using `internalChecksFilter: 'strict'` and it causes problems in Gitlab.

**Comment by rarkins** [comment]
See https://docs.renovatebot.com/configuration-options/#statuschecknames and try setting it to `null` to disable it

**Reply by fabb** [reply]
Thank you! I‘ve enabled this and didn’t see the `renovate/stability-days` check since!

**Reply by Bassadin** [reply]
How do I set it to null? The JSON schema validation says `Incorrect type. Expected "object".` if I just set it to `null` or `"null"`

**Reply by jamietanna** [reply]
I believe it's that you need to set:

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "statusCheckNames": {
    "minimumReleaseAge": null
  }
}
```

If that doesn't work, please raise a fresh Discussion, thanks! (this should've been closed and locked already)
